javalib/src/main/scala/java/time/chrono/ChronoLocalDate.scala

```diff
@@ -0,0 +1,124 @@
+package java.time.chrono
+
+import java.time.LocalDate
+import java.time.temporal._
+import java.util.Comparator
+
+trait ChronoLocalDate
+    extends Temporal with TemporalAdjuster with Comparable[ChronoLocalDate] {
+  import ChronoField._
+
+  def getChronology(): Chronology
+
+  def getEra(): Era = getChronology().eraOf(get(ERA))
+
+  def isLeapYear(): Boolean = getChronology().isLeapYear(get(YEAR))
+
+  def lengthOfMonth(): Int
+
+  def lengthOfYear(): Int
+
+  def isSupported(field: TemporalField): Boolean = field match {
+    case _: ChronoField if field.isDateBased => true
+
```

Comment:
Same for several other cases where condition and expression are on the same line.